odl/operator/default_ops.py

```diff
@@ -913,9 +812,9 @@ def __str__(self):
 
 class ResidualOperator(Operator):
 
-    """Operator that calculates the residual ``op(x) - vec``.
+    """Operator that calculates the residual ``op(x) - y``.
 
-    ``ResidualOperator(op, vector)(x) <==> op(x) - vec``
+    ``ResidualOperator(op, y)(x) <==> (x --> op(x) - y)``
```

Comment:
> dislike this syntax change.

What exactly? The `y <-> vector` change? I can revert that. Or do you mean something else? Note that the old one is internally inconsistent (`vector` and `vec`), and also inconsistent with other operators (some use `x --> something`, this one doesn't).